### ISSUE TITLE
Automatische Anrede

### ISSUE THREAD

**Issue by iradraconis** [jlawyerorg/j-lawyer-org#886]
Kann nicht das Feld für die Anrede automatisch gefüllt werden durch Beziehung der Daten aus der Adresse? Das würde bei der Anlage der Beteiligten einige Arbeit sparen. Oder aber die Anrede wird direkt bei der Adresseingabe per dropdown ausgewählt. Würde m.E. den Reiter "Textbausteine" obsolet machen

**Comment by j-dimension** [comment]
Ist mir nicht ganz klar. Wie soll erkannt werden, ob es Sehr geehrter Herr oder Frau sein soll?

**Comment by iradraconis** [comment]
Wenn bei der Adresseingabe direkt "Herr" oder "Frau" angegeben wird, könnte das Programm einen Vorschlag machen. So macht es mein bisheriges Kanzleiprogramm. Aktuell gibt es noch kein Feld für Herr oder Frau, das weiß ich :)

**Comment by iradraconis** [comment]
es müsste aber gar nicht erkannt werden, wenn das dropdown mit der Anrede direkt bei der Adresseingabe ausgewählt werden könnte. Würde wie gesagt den Reiter Textbausteine entbehrlich machen. Oder hat der noch einen anderen Zweck?

**Comment by j-dimension** [comment]
Schau mal die neue Adressverwaltung in der Onlinedemo an, da ist viel passiert. :-)

**Comment by j-dimension** [comment]
Ich nehme das Ticket in die 1.14

**Comment by j-dimension** [comment]
Wird hier noch etwas benötigt? Würde das Ticket sonst schließen.

**Comment by Dickmaulruessler** [comment]
> es müsste aber gar nicht erkannt werden, wenn das dropdown mit der Anrede direkt bei der Adresseingabe ausgewählt werden könnte. Würde wie gesagt den Reiter Textbausteine entbehrlich machen. Oder hat der noch einen anderen Zweck?

Ja, ich verstehe den Gedanken, der hinter dem gesonderten Reiter "Textbausteine" steht - optimal finde ich das aber irgendwie auch nicht. Ich hätte das eher unter "Person" vermutet, zumal man dort auch "Anrede" auswählen kann...
